### ISSUE TITLE
ModuleNotFoundError: No module named 'mmpose.datasets.dataset_info'

### ISSUE THREAD

**Issue by TaylorHere** [open-mmlab/mmdeploy#455]
Thanks for your bug report. We appreciate it a lot.

**Checklist**

1. I have searched related issues but cannot get the expected help.
2. I have read the [FAQ documentation](https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/faq.md) but cannot get the expected help.
3. The bug has not been fixed in the latest version.

**Describe the bug**

**Reproduction**

1. What command or script did you run?

```shell
python tools/deploy.py \                                                                                                                                     
configs/mmpose/pose-detection_static.py \
hrnet_w48_coco_384x288_udp.py \
hrnet_w48_coco_384x288_udp-0f89c63e_20210223.pth \
mmdeploy/demo/resources/human-pose.jpg \
--work-dir /tmp
```

2. Did you make any modifications on the code or config? Did you understand what you have modified?
no

**Environment**

1. Please run `python tools/check_env.py` to collect necessary environment information and paste it here.

mmdeploy git#v0.4.0
mmpose git#0.15.0
mmcv-full==1.3.18

3. You may add addition that may be helpful for locating the problem, such as
    - How you installed PyTorch [e.g., pip, conda, source]
    - Other environment variables that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)

```shell
2022-05-10 10:31:38,832 - mmdeploy - INFO - torch2onnx start.
load checkpoint from local path: /home/taylor/下载/hrnet_w48_coco_384x288_udp-0f89c63e_20210223.pth
2022-05-10:10:31:44,root ERROR    [utils.py:43] No module named 'mmpose.datasets.dataset_info'
Traceback (most recent call last):
  File "/home/taylor/Projects/mmdeploy/mmdeploy/utils/utils.py", line 38, in target_wrapper
    result = target(*args, **kwargs)
  File "/home/taylor/Projects/mmdeploy/mmdeploy/apis/pytorch2onnx.py", line 105, in torch2onnx
    data, model_inputs = task_processor.create_input(img, input_shape)
  File "/home/taylor/Projects/mmdeploy/mmdeploy/codebase/mmpose/deploy/pose_detection.py", line 139, in create_input
    from mmpose.datasets.dataset_info import DatasetInfo
ModuleNotFoundError: No module named 'mmpose.datasets.dataset_info'
2022-05-10 10:31:45,452 - mmdeploy - ERROR - torch2onnx failed.
```

If applicable, paste the error trackback here.

```none
A placeholder for trackback.
```

**Bug fix**

If you have already identified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!

**Comment by hanrui1sensetime** [comment]
Hi, let me help you to solve this issue.

**Comment by hanrui1sensetime** [comment]
Hi, @TaylorHere, have you tried to upgrade the mmcv version and mmpose version? If not, you can have a try to upgrade mmcv to 1.4.0+, and mmpose upgrade to 0.24.0.

**Comment by TaylorHere** [comment]
@hanrui1sensetime hi I solved this issue with mmdeploy@v0.4.0 mmpose@dev-0.25 mmcv-full==1.5.0

**Comment by TaylorHere** [comment]
If there is a support matric will be helpful for users
